src/amber/server/settings.cr

```diff
@@ -1,26 +1,63 @@
 require "./cluster"
-require "./configuration"
 require "./ssl"
 
 module Amber
   class Settings
     include Amber::DSL::Server
-    property port_reuse : Bool
-    property log : ::Logger
-    property color : Bool = true
-    property process_count : Int32
-    property secret_key_base : String
-    property port : Int32
-    property name : String
-    property host : String
-    property ssl_key_file : String? = nil
-    property ssl_cert_file : String? = nil
-    property redis_url = ""
+    property log : ::Logger = Logger.new(STDOUT)
+
+    property colorize_logging : Bool = true
     property database_url = ""
-    property session : Hash(Symbol, Symbol | String | Int32)
+    property host : String = "localhost"
```

Comment:
We shouldn't be defining defaults in both property definitions and yaml mappings. I'd vote for putting all defaults in yaml mappings.

Reply:
Completely agree.